src/classes/GE_PaymentServices.cls

```diff
@@ -132,11 +132,9 @@ public with sharing class GE_PaymentServices {
 
             generator.writeStartObject();
             generator.writeStringField('iss', 'ApiKeyManager'); // Issuer
-            generator.writeNumberField('iat', getEpochTimestamp(Datetime.now())); // Issued at
             generator.writeNumberField('exp', getExpirationDate(1)); // Expires at
-            generator.writeStringField('aud', 'www.example.com'); // Audience
-            generator.writeStringField('sub', 'cbaxter'); // Subject
-            generator.writeStringField('sfdo.id', '00D6g0000000000EAA'); // Org ID
+            generator.writeStringField('sub', UserInfo.getUserName()); // Subject
```

Comment:
@ptewson-sfdo I think we can call the `getUsername()` method that we created instead of calling the `getUserName` method on the UserInfo class directly.

Reply:
Hmm. We are in the inner class here, so I don't think it's super straightforward to call the method on the parent class?

Reply:
Yeah, unfortunately I think so as well. Might be cleaner to just call `UserInfo.getUsername()` in the two lines directly and clear out the outer `getUserName()` method.

Reply:
Yeah I think you both are right, as long as we take out the outer one, if we wanted to call the outer one, we would need to instantiate the parent class.

Reply:
Updated!